### PR TITLE
SNMP Collector

### PR DESCRIPTION
**Pull request by weslayer**

### CODE REVIEW

**Comment by weslayer** [comment]
Resolves #29

File: collector/server.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+from fastapi.middleware.cors import CORSMiddleware
+import prometheus_client
+from pysnmp.hlapi import *
+import uvicorn
+
+# from modules.metrics import (
+#     page_count_metric,
+#     ink_percent_metric,
+#     snmp_req_duration
+# )
```
Comment (evanugarte) [inline collector/server.py]:
```suggestion
```

File: collector/server.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                            snmp_error.labels(name=oid.metric_name).set(int(res[1] == 3))
+                            continue
+
+                        snmp_metric.labels(name=oid.metric_name).set(res[1])
+                        if oid.metric_name == "ink_level":
+                            ink_level = res[1]
+                        elif oid.metric_name == "ink_capacity":
+                            ink_cap = res[1]
+                    if ink_cap:
+                        snmp_metric.labels(name="ink_percent").set(ink_level/ink_cap)
```
Comment (evanugarte) [inline collector/server.py]: i dont think we need this metric we can just have grafana calculate it for us

Comment (evanugarte) [inline collector/server.py]:
```suggestion
                        snmp_metric.labels(name=oid.metric_name).set(res[1])
```

**Review by evanugarte — APPROVED** [review]

**Pull request merged**